javascript/javascript1/week4/script.js

```diff
@@ -0,0 +1,131 @@
+let info = [{       // User name array
+    name: ""
+}]
+
+let todolist = []       // Todo list array
+
+function getReply(command) {
+    let arr = []        // Users inputs arrays
+    command1 = command.split(' ')
+    arr.push(command1)
```

Comment:
The `.split` method creates an array from a string.
So you don't have to have two layers here, you can simply do:
`const commandWords = command.split(" ");`
and then use the `commandWords` array in the rest of the file. 

See further comments for examples

Reply:
If you want to make it able to handle capitalized and lowercase letters, you can transform it first:
`const commandWords = command.toLowerCase().split(" ");`